src/applications/facility-locator/actions/index.js

```diff
@@ -208,126 +195,58 @@ export const genBBoxFromAddress = query => {
     dispatch({ type: SEARCH_STARTED });
 
     // commas can be stripped from query if Mapbox is returning unexpected results
-    /**
-      New sdk requires types to be an array otherwise string
-     */
-    let types = environments.isStaging()
-      ? ['place', 'region', 'postcode', 'locality']
-      : 'place,region,postcode,locality';
+    let types = ['place', 'region', 'postcode', 'locality'];
     // check for postcode search
     if (query.searchString.match(/^\s*\d{5}\s*$/)) {
```

Comment:
Could we abstract this into a variable to make it a bit more readable?

Something like:
```js
// ...
const isPostcode = query.searchString.match(/^\s*\d{5}\s*$/);

if (isPostcode) {
// ...
```